src/dredd-command.js

```diff
@@ -254,6 +254,11 @@ ${packageData.name} v${packageData.version} \
         this._processExit(1);
       });
 
+      // Fixes https://github.com/apiaryio/dredd/issues/1030
+      process.stdin.on('error', (err) => {
+        logger.error(`Error in stdin: ${err}`);
+      });
```

Comment:
This is global setting (`process.stdin`) in a section of code relevant to spawning a server process. As the server process isn't the only child process Dredd spawns (server process & hook handler) and the setting influences both child processes and the Dredd process as whole, I wonder whether this shouldn't be on a common, neutral place.

Maybe `DreddCommand#constructor`?

Reply:
Addressed in https://github.com/apiaryio/dredd/pull/1035/commits/b205cc9a65bb447d1ac231971a5583c9ce633878.